Assets/Scripts/CSharp7/CSharp7.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class CSharp7 : MonoBehaviour
+{
+    [SerializeField] private int _skipCount;
+    [SerializeField] private int _takeCount;
+
+    private int _vectorCount = 5;
+    private List<int> xPositions;
+
+    public void Start()
+    {
+        var list = new List<Vector3>();
```

Comment:
You have created an empty list.
But did not create random vectors, as it is written in TK

Reply:
+?